src/styles/index.scss

```diff
@@ -1,3 +1,125 @@
-body {
+$main-accent: #060B35;
+$secondary: #616070;
+$blue-accent: #00ACDC;
+$white: #FFF;
+$border-color: #F3F3F3;
+
+$star-size: 16px;
+
+.page__body {
   margin: 0;
+  font-family: Roboto, sans-serif;
+}
+
+.product-card {
```

Comment:
it's a separate bem block - move it to its own file

Reply:
Not fixed